src/Arcus.Messaging.Pumps.ServiceBus/Configuration/AzureServiceBusMessagePumpSettings.cs

```diff
@@ -34,6 +35,7 @@ public class AzureServiceBusMessagePumpSettings
         /// <exception cref="ArgumentException">
         ///     Thrown when the <paramref name="getConnectionStringFromConfigurationFunc"/> nor the <paramref name="getConnectionStringFromSecretFunc"/> is available.
         /// </exception>
+        /// TODO: remove 'old' workings after the background jobs package is updated with the new messaging package.
         [Obsolete("Use the other constructor overload with the build-in '" + nameof(ServiceBusEntityType) + "' enumeration")]
```

Comment:
This is a 1.0 release, please just remove it

Reply:
Removed, resolving.